src/components/PoolPositionCard/PoolPositionCard.tsx

```diff
@@ -13,11 +12,11 @@ import { DoubleCurrencyLogo } from 'components';
 import { formatAPY, getAPYWithFee, getOneYearFee } from 'utils';
 import PoolPositionCardDetails from './PoolPositionCardDetails';
 import 'components/styles/PoolPositionCard.scss';
+import { Trans, useTranslation } from 'react-i18next';
 
 const PoolPositionCard: React.FC<{ pair: Pair }> = ({ pair }) => {
+  const { t } = useTranslation();
   const [bulkPairData, setBulkPairData] = useState<any>(null);
-  const { breakpoints } = useTheme();
-  const isMobile = useMediaQuery(breakpoints.down('xs'));
```

Comment:
Why did we remove mobile

Reply:
Updated css for mobile responsive and removed as this is unnecessary